### DISCUSSION TITLE
How is author supposed to work?

### DISCUSSION THREAD

**Discussion by Ravencentric** [retypeapp/retype#491] [Q&A] [answered]
for example i've added an author here: https://github.com/Snaacky/thewiki/blob/master/docs/index.md
But I dont see anything in retype: https://thewiki.moe/
Am I doing something wrong?

**Comment by geoffreymcgill** [comment, marked as answer]
The [`author`](https://retype.com/configuration/page/#author) docs demonstrate the options. 

Something as basic as the following will create the `author` details on the page. 

```yml
---
authors: [Moe]
---
```

Hope this helps.

**Comment by Ravencentric** [comment]
Thank you
